pilot/pkg/model/service.go

```diff
@@ -1024,6 +1024,8 @@ func (i WorkloadInfo) ResourceName() string {
 type WorkloadAuthorization struct {
 	// LabelSelectors for the workload. Note these are only used internally, not sent over XDS
 	LabelSelector
+	// RootNamespace indicates if this is a root namespace policy whose label selector applies across namespaces
+	RootNamespace bool
```

Comment:
Authorization already has a namespace field.

Reply:
Yeah but we don't know if it's the root namespace without mesh config. though we can probably query it when we need it I suppose

Reply:
Actually we know mesh config in `buildWorkloadPolicies`

Reply:
:+1: done